src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -79,21 +79,19 @@ describe('components/Popups/ReplyField', () => {
 
     describe('getVirtualElement()', () => {
         beforeEach(() => {
```

Comment:
We should share the spy or declare it in each test rather than both.

```
const getSelectionSpy = jest.spyOn(window, 'getSelection');

//..later

getSelectionSpy.mockReturnValueOnce({
    focusNode: null,
});
```